src/screens/Settings/ChangePassword.js

```diff
@@ -37,21 +41,34 @@ const initialValues = {
 export default class ChangePassword extends Component {
   static navigationOptions = () => ({ title: "Change Password" });
 
-  state = { ...initialValues };
-
-  onSubmit = async (values) => {
+  onSubmit = async (values, { resetForm }) => {
     let response;
+    let authResponse;
```

Comment:
I think these can just be inline `const`s

Reply:
Ah, this is a habit from doing promise-based api calls for years😅 it's to prevent breaking errors from getting thrown in case the var gets acted on before the thing that sets it gets returned. Can change that in the next PR, should be fine since we're using ES6...